src/components/governance/PageProposal.vue

```diff
@@ -189,6 +200,19 @@ export default {
     submittedAgo({ proposal } = this) {
       return moment(new Date(proposal.submit_time)).fromNow()
     },
+    endDate({ proposal } = this) {
+      return moment(proposal.voting_end_time).format("MMMM Do YYYY, HH:mm")
+    },
+    displayEndDate({ proposal } = this) {
+      if (
+        proposal.proposal_status !== "DepositPeriod" &&
+        proposal.total_deposit[0].amount >= 521000000
```

Comment:
We should use the proper parameter from the `governanceParameters` module instead of hardcoding this value here.

Reply:
Could you please point me in the right direction to find the dynamic value for the minimum deposit amount before a proposal enters the voting period? I explored the getters but was unsuccessful.

Reply:
I do see the correct Getter. I initially ruled it out due to it giving the wrong value but it is the test value for min. deposits in development.